tests/integration/calculator_windows.txt

```diff
@@ -0,0 +1,17 @@
+nose.config: INFO: Ignoring files matching ['^\\.', '^_', '^setup\\.py$']
+\u529f\u80fd: \u7d2f\u52a0\u6570\u503c
+
+  \u4f5c\u4e3a\u4e00\u4f4d\u4e0d\u4f1a\u6570\u5b66\u7684\u7528\u6237
```

Comment:
Just so you know, Windows has no escape codes in the output. If you print this file to the console, it will have no colors. Windows sets up colors by special API calls. Colorama removes ANSI escape codes from the output and issues those calls, so there's no trace in the output.

Reply:
Ah right, thanks, I didn't know that :P never really written much OS level code before (or code that interacts with the OS anyway).